charts/scalardl-audit/files/scalardl-audit_grafana_dashboard.json

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
This is a copy from `charts/scalardl-audit/files/scalardl_grafana_dashboard.json` as well.
The file name is changed for consistency.